packages/api/tests/userFarm.test.js

```diff
@@ -1013,6 +1022,55 @@ describe('User Farm Tests', () => {
       });
     });
 
+    describe('Set wage_do_not_ask_again', () => {
+      const testWithAdminRole = async (userRoleId, targetUserRoleId, done) => {
+        const { user: owner, farm } = await setupUserFarm({ role_id: userRoleId });
+        const worker = await createUserFarmAtFarm({ role_id: targetUserRoleId }, farm);
+        const target_user_id = worker.user_id;
+        setWageDoNotAskAgainRequest(
+          { user_id: owner.user_id, farm_id: farm.farm_id },
+          target_user_id,
+          async (err, res) => {
+            expect(err).toEqual(null);
+            expect(res.status).toBe(200);
+            const updatedUserFarm = await userFarmModel
+              .query()
+              .where('farm_id', farm.farm_id)
+              .andWhere('user_id', target_user_id)
+              .first();
+            expect(updatedUserFarm.wage_do_not_ask_again).toEqual(true);
+            done();
+          },
+        );
+      };
+
+      test('Owner should be able to set wage_do_not_ask_again', async (done) => {
+        testWithAdminRole(1, 3, done);
+      });
+
+      test('Manager should be able to set wage_do_not_ask_again', async (done) => {
+        testWithAdminRole(2, 3, done);
+      });
+
+      test('EO should be able to set wage_do_not_ask_again', async (done) => {
+        testWithAdminRole(5, 3, done);
+      });
+
+      test('Farm worker should not be able to set wage_do_not_ask_again', async (done) => {
```

Comment:
Same as other test - since this is so similar to testWithAdminRole() could it be simply testWithRole() and split expect statements ?